app-text/xmlto/xmlto-0.0.28-r4.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit eutils
```

Comment:
Also, is eutils needed?

Reply:
Thanks for pointing me here. Line removed.